br/pkg/backup/schema.go

```diff
@@ -79,8 +96,9 @@ func (ss *Schemas) AddSchema(
 }
 
 // BackupSchemas backups table info, including checksum and stats.
-func (ss *Schemas) BackupSchemas(
+func BackupSchemas[T SchemaIterator](
```

Comment:
Please don't move interface types to type parameter in go. The current go compiler won't specialize this function for every type but just [dispatch the parameter at runtime](https://go.dev/blog/when-generics#dont-replace-interface-types-with-type-parameters) -- we cannot take good from compile time polymorphism.

Reply:
done